subsys/bluetooth/audio/Kconfig.has

```diff
@@ -70,7 +27,9 @@ config BT_HAS_PRESET_SUPPORT
 
 if BT_HAS_PRESET_SUPPORT
 
-if BT_HAS_HEARING_AID_BINAURAL
+# Note: BT_HAS_IDENTICAL_PRESET_RECORDS & BT_HAS_PRESET_SYNC_SUPPORT below
+# are only relevant for binaural hearing aids.
+# TODO: Move from Kconfig to bt_has_register_param in a subsequent PR
```

Comment:
Apart form the note, I think it's good to ignore those Kconfig option values in `has.c` if non-binaural type is registered